vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -24,4 +24,9 @@ def raw_update_stack(options)
       connection.update_stack(stack, planning_data)
     end
   end
+
+  def update_ready?(state)
+    # Update is possible only when in complete or failed state, otherwise API returns exception
+    state.end_with?("_COMPLETE", "_FAILED")
```

Comment:
If possible, this should log what's "in progress" if the state is `<something>_IN_PROGRESS`.  Are there details included in the state information that could be logged to know why the update is not ready?